DevelopmentConventions.md

```diff
@@ -2,8 +2,8 @@
 
 This document describes intended and preferred conventions
 for development and maintenance of the Redistricting a Region software product.
-* [Agile Principles](/developmentConventions.md#Agile-Principles)
-* [Coding Convention Guide](/developmentConventions.md#Coding-Convention-Guide)
+* [Agile Principles](/DevelopmentConventions.md#Agile-Principles)
+* [Coding Convention Guide](/DevelopmentConventions.md#Coding-Convention-Guide)
```

Comment:
Why are these changes part of this PR?
Are you sure that this is PR was rebased using the current project master brach?

Reply:
I was having issues with the rebase. Locally I only had one commit but the pull request was showing 6 commits. When I tried to squash the commits they were unavailable to squash. I think part of the problem was when we were merging pulls into our team master instead of a branch.  I decided to do a --soft reset. It worked for my commits but I think it went back one commit to far and picked up your commit where you modified Devlpoments.md

Reply:
I have the extra changes problem fixed so the commit does not include the extra change